source/dub/commandline.d

```diff
@@ -1232,6 +1233,7 @@ class DescribeCommand : PackageBuildCommand {
 		settings.buildType = m_buildType;
 		settings.compiler = m_compiler;
 		settings.filterVersions = m_filterVersions;
+		settings.buildSettings.options |= m_buildSettings.options & BuildOption.lowmem;
```

Comment:
What exactly does this change in the describe command? I would imagine because m_buildSettings isn't used at all, that it would miss other stuff too.

Reply:
You can get lowmem by running `dub describe --data=options --data-list`.

>I would imagine because m_buildSettings isn't used at all, that it would miss other stuff too.

I suspect that the fact that m_buildSettings is not being used at all is a bug...
For example, the following does not show the expected debug version(TEST_VER) because of this:
```
dub describe --data=debug-versions --data-list -d=TEST_VER
```

Reply:
made an issue #1911